src/js/libcs/RenderBackends.js

```diff
@@ -851,26 +851,36 @@ function exportWith(Context, wnd) {
             var blob = csctx.toBlob();
             releaseExportedObject();
             exportedCanvasURL = window.URL.createObjectURL(blob);
-            wnd.location.href = exportedCanvasURL;
+
+            downloadHelper(exportedCanvasURL);
+            window.URL.revokeObjectURL(exportedCanvasURL);
```

Comment:
This is overlapping functionality of `releaseExportedObject`. When I wrote that, I was unsure about when we can revoke an object URL. To err on the side of caution, I decided to only revoke URLs when the next export is underway, so that we have at most one object active at any point, but the most recently exported object stays alive until superseded.

If you are certain that revoking the URL immediately is safe, and perhaps even have some documentation supporting that claim, then you can drop `releaseExportedObject` (both the call and the implementation) and make `exportedCanvasURL` local to this scope here.